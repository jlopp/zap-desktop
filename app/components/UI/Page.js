import React from 'react'
import { Flex } from 'rebass'

/**
 * @render react
 * @name Page
 * @example
 * <Page>Some content</Page>
 */
const Page = props => (
  <Flex
    {...props}
    as="article"
    alignItems="stretch"
    width="950"
    bg="white"
    css={{ height: '600px', 'min-height': '700px', 'min-width': '950px' }}
  />
)

export default Page
