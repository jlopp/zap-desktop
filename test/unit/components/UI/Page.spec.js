import React from 'react'
import Page from 'components/UI/Page'
import renderer from 'react-test-renderer'

describe('component.UI.Page', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Page />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
