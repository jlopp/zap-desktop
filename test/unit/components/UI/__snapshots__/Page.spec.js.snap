// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.Page should render correctly 1`] = `
.c0 {
  width: 950;
  background-color: white;
  height: 600px;
  min-height: 700px;
  min-width: 950px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

<article
  className="c0"
  width="950"
/>
`;
